### PR TITLE
Add debian bookworm packages

### PR DESCRIPTION
**Pull request by mosfet80**
Please add the following dependency to the rosdep database.

## Package name:

* libignition-cmake-dev
* libignition-math-dev
* libboost-date-time1.81.0
* libboost-python1.81.0
* libboost-random1.81.0
* libboost-thread1.81.0
* libogre-1.12-dev

## Purpose of using this:

The added dependencies are needed to compile ROS 2 on Debian "bookworm".

## Links to Distribution Packages

- Debian: https://packages.debian.org/
  - https://packages.debian.org/bookworm/libignition-cmake-dev
  - https://packages.debian.org/bookworm/libignition-math-dev
  - https://packages.debian.org/bookworm/libboost-date-time1.81.0
  - https://packages.debian.org/bookworm/libboost-python1.81.0
  - https://packages.debian.org/bookworm/libboost-random1.81.0
  - https://packages.debian.org/bookworm/libboost-thread1.81.0
  - https://packages.debian.org/bookworm/libogre-1.12-dev

### CODE REVIEW

**Review by emersonknapp — APPROVED** [review]

File: rosdep/base.yaml [hunk 1]
```diff
@@ -4400,6 +4407,8 @@ libogre:
     jessie: [libogre-1.9.0]
     stretch: [libogre-1.9.0v5]
     wheezy: [libogre-dev]
+    bookworm: [libogre-1.12-dev]
+
```
Comment (emersonknapp) [inline rosdep/base.yaml:4411]:
```suggestion
    bookworm: [libogre-1.12-dev]
```

**Comment by nuclearsandwich** [comment]
@mosfet80 can you please fill out the pull request template with some basic details and intent. Since the changes are all updates to existing keys it's not an impossible review but it is made more challenging by the lack of information in the PR description.

**Review by nuclearsandwich — CHANGES_REQUESTED** [review]
There are issues in the PR checks to address before this is ready to review.

**Review by nuclearsandwich — DISMISSED** [review]
Thank you for iterating on the PR description. I made some additional formatting and phrasing edits to it and it is now in much better shape.

Can you please address the issues reported in the CI checks. For example:
```
 checking for item order...
  ERR: list out of alphabetical order line 1969.  'bookworm' should come before 'buster'
  ERR: list out of alphabetical order line 2076.  'bookworm' should come before 'buster'
  ERR: list out of alphabetical order line 2634.  'bookworm' should come before 'buster'
  ERR: list out of alphabetical order line 2669.  'bookworm' should come before 'buster'
  ERR: list out of alphabetical order line 2778.  'bookworm' should come before 'buster'
  ERR: list out of alphabetical order line 2810.  'bookworm' should come before 'buster'
  ERR: list out of alphabetical order line 2913.  'bookworm' should come before 'buster'
  ERR: list out of alphabetical order line 4410.  'bookworm' should come before 'wheezy'
  ```

**Review by clalancette — APPROVED** [review]

**Pull request merged**